test/composition.jl

```diff
@@ -40,10 +40,10 @@ using Test, LinearMaps, LinearAlgebra, SparseArrays
     @test @inferred !issymmetric(FC'FC)
     @test @inferred ishermitian(FC'FC)
     @test @inferred ishermitian(FC'*H*FC)
-    @test @inferred isposdef(transpose(F) * F * 3)
-    @test @inferred isposdef(transpose(F) * 3 * F)
-    @test @inferred !isposdef(-5*transpose(F) * F)
-    @test @inferred isposdef((M * F)' * M * 4 * F)
+#   @test @inferred isposdef(transpose(F) * F * 3)
+#   @test @inferred isposdef(transpose(F) * 3 * F)
+#   @test @inferred !isposdef(-5*transpose(F) * F)
+#   @test @inferred isposdef((M * F)' * M * 4 * F)
```

Comment:
Let's change this to `issymmetric` to check for the destructuring logic:
```suggestion
    @test @inferred issymmetric(transpose(F) * F * 3)
    @test @inferred issymmetric(transpose(F) * 3 * F)
    @test @inferred !isposdef(-5*transpose(F) * F)
    @test @inferred ishermitian((M * F)' * M * 4 * F)
```